link_traffic/munin/link_traff2json_

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
+#%# family=auto
+#%# capabilities=autoconf suggest
+
+import socket
+import sys
+import os
+import time
+import datetime
+import csv
+import json
+
+# path to configuration file 
+CONFIG_FILE = '../config.cfg'
+# path to data saved by link_traffic module 
+SAVED_DATA = '/var/run/libtrap/saved_data'
+# temporary file with downloaded data
+data_temp_file = "/tmp/munin_link_traffic_data.txt"
+# path to munin socket 
+SOCKET_PATH = '/var/run/libtrap/munin_link_traffic'
+# munin refresh interval
+interval = 60
+time_mon = "{:%Y-%m-%d %H:%M:%S}".format(datetime.datetime.today())
+time_mon_st = 0
+#=======================================================================
+"""!@fn receiveData 
+  @brief function that connects to file with saved data
+  @return string of data set from socket"""
+def receiveData():
+   sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+   try:
+      sock.connect(SOCKET_PATH)
+   except socket.error as msg:
+      sys.stderr.write(SOCKET_PATH + " " + str(msg) + "\n")
+      sys.exit(1)
+
+   data = ""
+   while True:
+      tmp = sock.recv(4096)
+      data += tmp
+      if tmp == "":
+         break;
+   
+   return data
+#=======================================================================
+"""!@fn getData 
+  @return list of strings (lines).
+   There should be 2 lines: the first one with header,
+   the second one with values. The data is received by
+   receiveData() and stored into a cache file data_temp_file.
+   If data_temp_file is new enough, data is loaded from the 
+   cache file instead of calling receiveData()."""
+def getData():
+    current_time = time.time()
+    last_time = 0
+    data = ""
+    try:
+        last_time = os.stat(data_temp_file).st_mtime
+    except:
+        sys.stderr.write("%s not found or cannot read info.\n" % data_temp_file)
+        pass
+    if (current_time - last_time) > interval:
+        data = receiveData()
+        sys.stderr.write("Reading from socket\n")
```

Comment:
Move this one line above (i.e. print the message before trying to read the data)